src/jobsautoreport/query.py

```diff
@@ -65,6 +69,14 @@ def _get_query_steps_by_name(
             }
         }
 
+    def query_equinix_project_usages(self, from_date: datetime, to_date: datetime) -> list[EquinixUsage]:
+        headers = {"X-Auth-Token": self._equinix_project_token}
+        url = f"https://api.equinix.com/metal/v1/projects/{self._equinix_project_id}/usages"
+        equinix_project_usages = requests.get(url=url, headers=headers).json()['usages']
```

Comment:
isn't the api paginated?

Reply:
I thought so too but I did not see it applies for usages